plugins/early_talker.js

```diff
@@ -64,3 +76,52 @@ exports.early_talker = function(next, connection) {
 
     setTimeout(function () { check(); }, pause);
 };
+
+
+/**
+ * Check if the ip is whitelisted
+ *
+ * @param  {Array} whitelist A list of ipaddr objects
+ * @param  {String} ip       The remote IP to verify
+ * @return {Boolean}         True if is whitelisted
+ */
+exports.ip_in_list = function (whitelist, ip) {
+    var ipobj = ipaddr.parse(ip);
+
+    for (var i = 0; i < whitelist.length; i++) {
+        try {
+            if (ipobj.match(whitelist[i])) {
+                return true;
+            }
+        } catch (e) {
```

Comment:
in cases where you're ignoring the caught error, rename the error from 'e' to 'ignore', making your intentions explicit.